### ISSUE TITLE
UserDetails Missing Accessible Control Grouping

### ISSUE THREAD

**Issue by johnhbenetech** [benetech/MathShare#794]
The grade check boxes on the teacher details form are all part of a single group with the title, “What grade do you work with?”.  However, this grouping is not programmatically indicated, and a screen reader user navigating through the page using the tab key will be unaware of the question they are being asked to answer. 

Suggested Solution 
 
Wrap the check boxes inside a < fieldset> element (one < fieldset> containing all of the check boxes, not one per box).  Add a < legend> as the first child element of the fieldset, and move the question text into it. 

Proposed Code Sample 
 
`<div className="row">  <fieldset>   <legend className={`${userDetails.descText} col-5`}>What grade do you work with?</legend>   … check boxes …  </fieldset>`

Aha! Link: https://diagramlabs.aha.io/features/B-124

**Comment by jscholes** [comment]
Fixed in https://github.com/benetech/MathShare/commit/b5032a9952af98a88dd5775e6526cc9c5f8cec08.
